### PR TITLE
develop: include Django in Python ORM

### PR DESCRIPTION
**Pull request by wd0517**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/14987
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by Icemap — APPROVED** [review]
LGTM

File: develop/dev-guide-choose-driver-or-orm.md [hunk 1]
```diff
@@ -273,6 +273,15 @@ For an example of using MySQL Connector/Python to build a TiDB application, see
...
+<div label="Django">
+
+Support level: **Full**
+
+[Django](https://docs.djangoproject.com/) is a popular Python web framework. We have developed an adapter `django-tidb` for Django to solve compatibility issues. For installation instructions, please refer to the [documentation](https://github.com/pingcap/django-tidb#installation-guide).
+
+For an example of using Django to build a TiDB application, see [Connect to TiDB with Django](/develop/dev-guide-sample-application-python-django.md).
+
+</div>
 <div label="SQLAlchemy">
```
Comment (Oreoxmt) [inline develop/dev-guide-choose-driver-or-orm.md:285]: L300:
```
For an example of using peewee to build a TiDB application, see [Connect to TiDB with peewee](/develop/dev-guide-sample-application-python-peewee.md#step-2-get-the-code).
```

File: develop/dev-guide-choose-driver-or-orm.md [hunk 2]
```diff
@@ -273,6 +273,15 @@ For an example of using MySQL Connector/Python to build a TiDB application, see
 ### Python ORM frameworks
 
 <SimpleTab>
+<div label="Django">
+
+Support level: **Full**
+
+[Django](https://docs.djangoproject.com/) is a popular Python web framework. To solve the compatibility issue between TiDB and Django, PingCAP has developed an adapter `django-tidb`. Installation instructions can be found in the [`django-tidb` documentation](https://github.com/pingcap/django-tidb#installation-guide).
```
Comment (Oreoxmt) [inline develop/dev-guide-choose-driver-or-orm.md:280]:
```suggestion
[Django](https://docs.djangoproject.com/) is a popular Python web framework. To solve the compatibility issue between TiDB and Django, PingCAP provides a TiDB dialect `django-tidb`. To install it, you can see the [`django-tidb` documentation](https://github.com/pingcap/django-tidb#installation-guide).
```

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**